app/controllers/application_controller.rb

```diff
@@ -211,6 +211,7 @@ def authorize_user_for_course
     @cud.errors.full_messages.each do |msg|
       flash[:error] += "<br>#{msg}"
     end
+    flash[:html_safe] = true
```

Comment:
See #1549 for context. Essentially, the PR escaped all flash messages by default (previously, we used `sanitize` which allowed unintended styling from user controlled strings) and set the "flag" `flash[:html_safe] = true` if we wish to display the message verbatim.

Touching up on some flashes which use `<br>` that I missed out on.